src/app/my/edit/page.tsx

```
@@ -26,6 +29,12 @@ export default function EditPage() {
         }
     }
     return <div className="h-full mt-8 w-[var(--content-width)] mx-auto">
+        <div className="flex w-full mb-5">
+            <button className="text-sm text-gray-500 flex items-center gap-2" onClick={() => router.push("/my")}>
+                <ArrowLeftIcon className="w-4 h-4" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)